openmetadata-service/src/test/java/org/openmetadata/service/resources/services/DashboardServiceResourceTest.java

```diff
@@ -85,8 +85,8 @@ void post_withoutRequiredFields_400_badRequest(TestInfo test) {
   void post_validService_as_admin_200_ok(TestInfo test) throws IOException, URISyntaxException {
     // Create dashboard service with different optional fields
     Map<String, String> authHeaders = ADMIN_AUTH_HEADERS;
-    SupersetConnection supersetConnection =
-        new SupersetConnection()
+    MetabaseConnection supersetConnection =
```

Comment:
switched dashboard from superset to metabase due to schema changes

Reply:
Metabase with SSO we'll need to support from DB as well right?

Reply:
I believe Metabase has API to authenticate with SSO [ref](https://www.metabase.com/docs/latest/api/ee/sso)